pages/index.html

```diff
@@ -4,9 +4,98 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <!-- <link rel="stylesheet" type="text/css" href="styles/style.css" /> -->
+  <link href="../styles/style.css" rel="stylesheet">
+  <title>Sana's Website</title>
 </head>
 <body>
+  <header>  
+    <h1>Sana's Swiftie Website</h1>
+
+    <div class="top">
+      <a href="about.html">About ME!</a>
+      <a href="portfolio.html"> My Portfolio</a>
+    </div>
+
+  </header>  
+
+
+<section>
+    <main>
```

Comment:
I don't think you need the section element here since the bulk of you content is already in the main element. I don't think it hurts, but I think it makes your HTML a little bloated by adding extra elements that aren't strictly necessary.